frontend-react/src/hooks/UseValueSets.test.ts

```diff
@@ -0,0 +1,111 @@
+import { rest } from "msw";
+import { renderHook } from "@testing-library/react-hooks";
+import { QueryClient } from "@tanstack/react-query";
+
+import { lookupTableServer } from "../__mocks__/LookupTableMockServer";
+import {
+    LookupTables,
+    ValueSet,
+    lookupTableApi,
+    LookupTable,
+} from "../network/api/LookupTableApi";
+import { QueryWrapper } from "../utils/CustomRenderUtils";
+
+import {
+    useValueSetsTable,
+} from "./UseValueSets";
+
+describe("useValueSetsTable", () => {
+    beforeAll(() => lookupTableServer.listen());
+    afterEach(() => lookupTableServer.resetHandlers());
+    afterAll(() => lookupTableServer.close());
+
+    test("returns expected values when fetching table version", async () => {
+        const { result, waitFor } = renderHook(
+            () => useValueSetsTable<ValueSet>(LookupTables.VALUE_SET),
+            { wrapper: QueryWrapper() }
+        );
+        await waitFor(() => !!result.current.valueSetArray.length);
+        const { name, system, createdAt, createdBy } =
+            result.current.valueSetArray[0];
+        expect(name).toEqual("sender_automation_value_set");
+        expect(createdAt).toEqual("now");
+        expect(createdBy).toEqual("test@example.com");
+        expect(system).toEqual("LOCAL");
+    });
+
+    test("returns expected values when using supplied table version", async () => {
+        const { result, waitFor } = renderHook(
+            () => useValueSetsTable<ValueSet>(LookupTables.VALUE_SET, 3),
+            { wrapper: QueryWrapper() }
```

Comment:
Is there a comparable setup to the [custom renderer pattern](https://testing-library.com/docs/react-testing-library/setup/#custom-render) we can use to abstract this wrapper setup?

Reply:
oh, yeah, it might be worth setting that up to use the AppWrapper by default. We'd still need some of the individual wrappers for when we want custom behavior (like turning off retries https://github.com/CDCgov/prime-reportstream/pull/6476/files#diff-a5b4cf7595f3c0e3d62866d65b3d69e271f15379db38680101b637af99d3fa35R73), but that could potentially be helpful.